base/src/test/java/io/spine/validate/FieldValidatorShould.java

```diff
@@ -0,0 +1,142 @@
+package io.spine.validate;
+
+import com.google.common.collect.ImmutableList;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.of;
+import static io.spine.test.Verify.assertNotEmpty;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Dmytro Dashenkov
+ */
+public abstract class FieldValidatorShould<V> {
+
+    /**
+     * Creates a {@link FieldValidator} for a repeated required validated field.
```

Comment:
`...for a repeated required field, which child items are also validated separately.`